h/static/scripts/time.js

```diff
@@ -6,43 +6,115 @@ var day = hour * 24;
 var month = day * 30;
 var year = day * 365;
 
+function lessThanThirtySecondsAgo(date, now) {
+  return ((now - date) < 30000);
```

Comment:
For consistency with the functions below I suggest writing these two constants as `N * 1000`